lib/go-parser/go-binary.ts

```diff
@@ -175,8 +175,17 @@ export function extractModuleInformation(
     throw Error("binary contains empty module info");
   }
 
-  const [, mainModuleLine, ...versionsLines] = mod.split("\n");
-  const [, name] = mainModuleLine.split("\t");
+  const [pathDirective, mainModuleLine, ...versionsLines] = mod.split("\n");
+  let [modDirective, name] = mainModuleLine.split("\t");
+  if (modDirective !== "mod") {
```

Comment:
do we even care about these binaries? do we have any vulns for the stdlib packages of Go?

Reply:
My initial thought was that sure, if the Go stdlib has a vulnerability we should also report these binaries so that customers could upgrade their Go env.

But looking into this, we don't return the stdlib packages as a dependency in the depgraph, so no we probably don't 😢 

(Though I think it would make sense to also return stdlib packages...)

Reply:
Technically, these binaries could also have vulnerabilities in their "own" code (which is not part of the standard library), which we might flag? I don't know :)

Reply:
I don't know if it's something we have in our DB. It's a good question, and I think we need to answer it first

Reply:
I think it would be good to show these binaries anyway, to have a more complete picture of what is in a container? With that in mind, it would be secondary whether or not we support vulns for those.

Reply:
Talked to a few different parties and we only support vulnerabilities in the Go standard library if we find the Go distribution (like the actual apk / deb / whatever package) in the container.

We don't support it from a scanned binary / SCM as of now. I'd argue that supporting this would make a lot of sense...

Either way, I still think my argument from above stands. What do you think?